### PR TITLE
Add linkinize extension

### PR DESCRIPTION
**Pull request by hazaveh**
## Description

Linkinize Raycast extension brings Linkinize integration right into your screen. quickly search and access your [Linkinize](https://linkinize.com) bookmarks with power of Raycast

## Screencast

Linkinize Raycast Extension

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
- Is there a reason why you don't use preferences for the username and password?

linkinize 2023-10-05 at 12 48 01

- The toast should be with `Failure` style instead

linkinize 2023-10-05 at 12 48 51

- You should use [metadata image tool](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) to generate metadata

File: extensions/linkinize/package.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  "icon": "icon.png",
+  "author": "hazaveh",
+  "categories": [
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Linkinize",
```
[the remark this answers is not in the document]
Comment (hazaveh) [reply extensions/linkinize/package.json]: Done ✅

File: extensions/linkinize/package.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      "subtitle": "Search through your Bookmarks",
```
[the remark this answers is not in the document]
Comment (hazaveh) [reply extensions/linkinize/package.json]: Done ✅

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
The login system feels a bit overengineered to me, maybe it's easier just to try with the current cached token, and if there is an error unset it and give the user a failure toast

**Comment by hazaveh** [comment]
> The login system feels a bit overengineered to me, maybe it's easier just to try with the current cached token, and if there is an error unset it and give the user a failure toast

@pernielsentikaer do you mean getting rid of the AuthComponent that displays the login issue completely? I'm afraid some users do not know how to set their credentials in the preferences.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
It's still lacking a bit since you can also disable commands in preferences, so it might not call the other commands.

Maybe you can use [useFetch](https://developers.raycast.com/utilities/react-hooks/usefetch), which also caches and gives you `isLoading` and `error` out of the box.

In case something goes wrong, I would simply show a toast and tell the user to correct the credentials instead.

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Pull request closed without merging**